public/app/plugins/datasource/elasticsearch/elastic_response.ts

```diff
@@ -324,12 +327,13 @@ export class ElasticResponse {
     }
   }
 
-  processHits(hits: { total: { value: any }; hits: any[] }, seriesList: any[]) {
+  processHits(hits: { total: { value: any }; hits: any[] }, seriesList: any[], target: any) {
     const hitsTotal = typeof hits.total === 'number' ? hits.total : hits.total.value; // <- Works with Elasticsearch 7.0+
 
     const series: any = {
-      target: 'docs',
+      target: target.refId,
       type: 'docs',
+      refId: target.refId,
```

Comment:
This was needed because otherwise the result would have been this: 
![image](https://user-images.githubusercontent.com/1170767/94133923-39979000-fe59-11ea-8eb2-62401e1875da.png)